pkg/controller/storagecluster/stork.go

```diff
@@ -60,6 +60,28 @@ const (
 	userVolumeNamePrefix    = "user-"
 )
 
+var (
+	storkServiceLabels = map[string]string{
+		"name": "stork",
+	}
+	storkDeploymentLabels = map[string]string{
+		"tier": "control-plane",
+	}
+	storkTemplateLabels = map[string]string{
+		"name": "stork",
+		"tier": "control-plane",
+	}
+	storkSchedulerDeploymentLabels = map[string]string{
+		"name":      storkSchedDeploymentName,
+		"tier":      "control-plane",
+		"component": "scheduler",
+	}
+	storkSchedulerTemplateLabels = map[string]string{
+		"tier":      "control-plane",
+		"component": "scheduler",
+	}
```

Comment:
This will conflict because of https://github.com/libopenstorage/operator/pull/524. I have merged the deployment and template labels for the scheduler. You can just use one set of labels here.